src/vsg/platform/macos/MacOS_Window.mm

```diff
@@ -813,6 +809,14 @@ void createApplicationMenus(void)
     _window.initialFirstResponder = _view;
     [_window makeFirstResponder:_view];
 
+    if (traits->fullscreen)
+    {
+        NSRect screenFrame = [[NSScreen mainScreen] frame];
+        [_window setBackgroundColor: NSColor.whiteColor];
```

Comment:
Is this line required as there is a similar but not quite identical code on line 798 above.

Reply:
Lines 814-815 are redundant and should be removed. It was blindly copy-pasted from the example.

<img width="1440" alt="Screenshot 2022-12-11 at 14 31 24" src="https://user-images.githubusercontent.com/88086187/206906720-c8fc4a94-cb03-4640-87d4-17a6ea2bb729.png">

Reply:
Thanks for the clarification, I've removed the redundant line and will now merge.

What is the story behind the screenshot?

Reply:
It's just vsgpagedlod example with static mapbox tiles and custom map style. By the way, vsg readers extract a file type from the file extension in the path from ```tileReader->imageLayer```. So it is not possible just to change the value of ```tileReader->imageLayer``` and experiment with 3rd party tile providers, like mapbox or HERE. mapbox and HERE tiles urls contain no file name with extension. It's confusing a little bit for newcomers. Possibly, it makes sense to extract file type from the server response instead of the file path.

Reply:
The vsg::tile ReaderWriter isn't intended to support all ways that tiles could be held on a server, it's a lightweight class that can handle some tile representations.  If a 3rd party source requires a query of supported formats from a server then I presume you'd need to formulate the query and way of mapping this to the final filename and extensions to sue then it'd need a new ReaderWriter that uses curl and as such would belong in vsgXchange.

Reply:
This type of discussion is best done within Discussions rather piggybacking off a PR.